extensions/data-workspace/src/common/constants.ts

```diff
@@ -20,6 +20,7 @@ export const EnterWorkspaceConfirmation = localize('dataworkspace.enterWorkspace
 export const WorkspaceContainsNotAddedProjects = localize('dataworkspace.workspaceContainsNotAddedProjects', "The current workspace contains one or more projects that have not been added to the workspace. Use the 'Open existing' dialog to add projects to the projects pane.");
 export const LaunchOpenExisitingDialog = localize('dataworkspace.launchOpenExistingDialog', "Launch Open existing dialog");
 export const DoNotShowAgain = localize('dataworkspace.doNotShowAgain', "Do not show again");
+export const ProjectsFailedToLoad = (numberOfProjects: number) => { return localize('dataworkspace.projectsFailedToLoad', "{0} project(s) failed to load. Please open console for more information", numberOfProjects); };
```

Comment:
I remembered a few years back David had to rephrase all the strings like project(s) to remove the (s). here you can simply say 'some projects failed to load...'